src/util/validate.py

```diff
@@ -15,6 +15,21 @@
 }
 
 
+def validate_assessment_id(assessment_id):
+    """Validate that the provided assessment_id is matching the valid assessment_id format. Example: RV1234.
+
+    Args:
+        assessment_id (string): Assessment identifier to validate.
+
+    Returns:
+        match: the result of a regular expression match.
+    """
+    pattern = re.compile("^RV([0-9]){4}")
+    match = pattern.match(assessment_id)
```

Comment:
You are re-compiling the regex every time you call the expression anyway, so you may as well use the one-liner syntax.  Also, the parentheses inside the regex are unnecessary, and you can replace `[0-9]` with the pithier `\d`:
```suggestion
    pattern = re.match(r"^RV\d{4}", assessment_id)
```

Reply:
See 0c1b4765449b645fc9b7b00779039ddee6410a92 for changes

Reply:
Hey @jsf9k, is this one resolved?

Reply:
Yep, sorry I overlooked it.